lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -246,6 +278,7 @@ class AzurePlatformSchema:
             validate=validate.Regexp(constants.GUID_REGEXP),
         ),
     )
+    cloud: Optional[Union[str, Dict[str, Any]]] = field(default=None)
```

Comment:
It can follow marketplace_image pattern. The type should be CloudEndpointsSchema here.

Reply:
updated